src/iexit.cpp

```diff
@@ -1,19 +1,31 @@
 /*** Copyright (c), The Regents of the University of California            ***
  *** For more information please refer to files in the COPYRIGHT directory ***/
+#include <irods/irods_exception.hpp>
+#include <irods/irods_service_account.hpp>
 #include <irods/rods.h>
 #include <irods/parseCommandLine.h>
 #include <irods/rcMisc.h>
```

Comment:
Since we'll be back here to remove the TODO below anyway, let's remove the Copyright lines and alphabetize these